pam/internal/adapter/nativemodel.go

```diff
@@ -584,14 +584,14 @@ func (m nativeModel) handleFormChallenge(hasWait bool) tea.Cmd {
 		}
 	}
 
-	instructions := "Insert '%[1]s' to cancel the request and go back"
+	instructions := "Enter '%[1]s' to cancel the request and go back"
```

Comment:
In the case of SSH, this results in the following:

```
$ ssh aadtest-testauthd@uaadtest.onmicrosoft.com@localhost
Enter 'r' to cancel the request and go back
(aadtest-testauthd@uaadtest.onmicrosoft.com@localhost) Enter your local password: 
```

That's already better than what we had before, but I think there's still room for improvement. I think this would be better:

```
$ ssh aadtest-testauthd@uaadtest.onmicrosoft.com@localhost
(aadtest-testauthd@uaadtest.onmicrosoft.com@localhost) Enter your local password (or enter 'r' to choose another authentication method):
```

The line is becoming quite long, but I still think it's better UX to have the "enter 'r'" part in the same line as the prompt. WDYT?

Reply:
The problem here is that we rely quite a bit on what the broker provides and so the text may be sub-optimal, reason why I was using two different sentences.

However, I think either way this is fine for now, although it has likely to be revised when translations will be a thing, since joining strings wouldn't work well in all the languages and even less in RTL ones.

Reply:
Ok, as per https://github.com/ubuntu/authd/pull/599#discussion_r1830009419 this can't be reliably done, unless switching the interface logic for SSH that I'd prefer to avoid unless for specific bits.

So, let's close this for now.